### ISSUE TITLE
New Basic Layer "nn.bias_add"

### ISSUE THREAD

**Issue by ZheyuYe** [apache/incubator-mxnet#17303]
## Description
I am working on the reproduce of ALBERT pretrained model from tensorflow into gluonnp (numpy nlp for specific), and I found that in the masking decoder of ALBERT model, the new learnable biases are used but the original embedding weights are loaded. I successful completed the implementation with the following code.
```
# only load the dense weights with a reinitial bias
decoder.add(nn.Dense(units=vocab_size, flatten=False, params=embed.collect_params('.*weight')))
```
However, I am still wondering if that possible to construct a **New Basic Layer** with only bias without considering weights for extension use, e.g. added as an item into the nn.HybridSequential stack.

## References
such as [**tf.nn.bias_add**](https://www.tensorflow.org/api_docs/python/tf/nn/bias_add)

**Comment by ptrendx** [comment]
You should be able to use `broadcast_add` there, no?

**Comment by sxjscience** [comment]
Given that we can use broadcasting, it should not be necessary to introduce `bias_add`. Also, in numpy, we can directly use the auto-broadcasting mechanism.
